Twig/RecommendationTwigExtension.php

```diff
@@ -78,11 +84,26 @@ public function __construct(
         $this->contentTypeService = $contentTypeService;
         $this->contentService = $contentService;
         $this->locationService = $locationService;
+        $this->userService = $userService;
         $this->localeConverter = $localeConverter;
         $this->authorizationChecker = $authorizationChecker;
         $this->tokenStorage = $tokenStorage;
         $this->session = $session;
-        $this->options = $options;
+        $this->options = $this->parseOptions($options);
+    }
+
+    /**
+     * @param array $options
+     *
+     * @return array
+     */
+    protected function parseOptions(array $options)
+    {
+        if (!empty($this->requestStack->getCurrentRequest()) && $this->requestStack->getCurrentRequest()->isSecure()) {
```

Comment:
Only small nitpick: can't you break this into multiple lines? There are a few places where it'd improve readability.